ros-rosapi/install.yaml

```diff
@@ -1,5 +1,4 @@
 - type: ros
   source:
-    type: git
-    url: https://github.com/RobotWebTools/rosbridge_suite.git
-    sub-dir: rosapi
+    type: system
```

Comment:
We want to run the source version. As we kept having issues with the released version. Or did you have the same problem and is it fixed now?